mcrouter/scripts/recipes/glog.sh

```diff
@@ -8,12 +8,16 @@ source common.sh
 
 if [ ! -d "$PKG_DIR/glog" ]; then
     git clone https://github.com/google/glog.git
-    cd "$PKG_DIR/glog" || die "cd fail"
-
-    autoreconf --install
-    LDFLAGS="-Wl,-rpath=$INSTALL_DIR/lib,--enable-new-dtags \
-             -L$INSTALL_DIR/lib $LDFLAGS" \
-        CPPFLAGS="-I$INSTALL_DIR/include -DGOOGLE_GLOG_DLL_DECL='' $CPPFLAGS" \
-        ./configure --prefix="$INSTALL_DIR" && \
-        make $MAKE_ARGS && make install $MAKE_ARGS
 fi
+
+cd "$PKG_DIR/glog" || die "cd fail"
+
+# Use a known compatible version
+# N.B. More recent versions (v0.5.0.rcx) failed to work with mcrouter.
+git reset --hard v0.4.0
```

Comment:
Here you use a `reset`, elsewhere you use a `checkout`. I don't know if there's reason to prefer one over the other or if the inconsistency even matters--just pointing it out.

Reply:
Thank's for pointing this out.  I've gone ahead and made these all consistent.